Jenkinsfile-Agent-Test

```diff
@@ -88,6 +88,12 @@ pipeline {
                                 sh 'bash test/plugin/run.sh --build_id=wl1_${BUILD_ID} spring-async-scenario'
                             }
                         }
+
+                        stage('netty-socketio 1.x (4)') {
+                            steps {
+                                sh 'bash test/plugin/run.sh --build_id=${BUILD_ID} netty-socketio-scenario'
```

Comment:
`--build_id=wl1_${BUILD_ID}` should replace --build_id=${BUILD_ID}

Reply:
resolved